pkg/log/logger.go

```diff
@@ -13,7 +13,7 @@ func ConfigLogger(level int8, syncer zapcore.WriteSyncer) logr.Logger {
 		level = -level
 	}
 	// The following settings is based on kyma community Improvement of log messages usability
-	//nolint:lll
+	//nolint:lll // link to logging documentation agreement
```

Comment:
Let's generally exclude urls from lll linter in the config with a regex like this:
```
    # Exclude lll issues for long lines starting with an url
    - linters:
        - lll
      source: "^// http "
```